docs/src/equations.md

```diff
@@ -211,17 +214,17 @@ This is discretized using the following
 ```math
 \frac{\partial}{\partial t} \rho \chi \approx
 - D_h[ \rho \chi (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))]
-- D^c_v\left[I^f(\rho) U^f\left(I^f(\boldsymbol{u}_h) + \boldsymbol{u}_v, \frac{\rho \chi}{\rho} \right) \right] .
+- D^c_v\left[WI^f(J, \rho) U^f\left(I^f(\boldsymbol{u}_h) + \boldsymbol{u}_v, \frac{\rho \chi}{\rho} \right) \right] .
 ```
 Currently the central reconstruction
 ```math
-- D^c_v[ I^f(\rho \chi) \boldsymbol{u}_v ]
+- D^c_v[ WI^f(J, \rho \chi) \boldsymbol{u}_v ]
 ```
 is treated implicitly.
 
 !!! todo
     The Jacobian computation should be updated so that the upwinded term
     ```math
-    - D^c_v\left[I^f(\rho) U^f\left(I^f(\boldsymbol{u}_h) + \boldsymbol{u}_v, \frac{\rho \chi}{\rho} \right) \right]
+    - D^c_v\left[WI^f(J, \rho) U^f\left(I^f(\boldsymbol{u}_h) + \boldsymbol{u}_v, \frac{\rho \chi}{\rho} \right) \right]
```

Comment:
Looks like it's missing the advective mass flux. It should parallel the energy flux once that is correct.